x-pack/plugins/apm/server/lib/errors/get_error_rate.ts

```diff
@@ -17,6 +17,9 @@ import {
 } from '../helpers/setup_request';
 import { rangeFilter } from '../../../common/utils/range_filter';
 
+// Regex for 5xx and 4xx
+const errorStatusCodeRegex = /5\d{2}|4\d{2}/;
+
 export async function getErrorRate({
```

Comment:
Can you add an api test for `GET /api/apm/services/{serviceName}/errors/rate` to cover this?